manageiq-gems-pending.gemspec

```diff
@@ -27,7 +27,9 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "aws-sdk",                 "~> 2.9.7"
   s.add_runtime_dependency "binary_struct",           "~> 2.1"
   s.add_runtime_dependency "bundler",                 ">= 1.8.4" # rails-assets requires bundler >= 1.8.4, see: https://rails-assets.org/
+  s.add_runtime_dependency "fog-openstack",           "~>0.1.22"
```

Comment:
Can you format this like every other line in the gemspec? With a space between the `>` and the version?

Reply:
doh!  Done.